R/QuantileTrimmedHistogram

```diff
@@ -0,0 +1,140 @@
+## Width calculation function
+calcWidth = function(query) { 
+  width(query)
```

Comment:
try to match the whitespace with the package you're submitting to -- in this case the package uses tabs, but you use 2 spaces.